src/maker/handlers.rs

```diff
@@ -657,12 +657,10 @@ impl Maker {
         let mut conn_state = self.ongoing_swap_state.lock()?;
         *conn_state = HashMap::default();
 
-        log::info!("initializing Wallet Sync.");
         {
-            let wallet_write = self.wallet.write()?;
-            wallet_write.save_to_disk()?;
+            let mut wallet_write = self.wallet.write()?;
+            wallet_write.sync_and_save()?;
```

Comment:
```suggestion
            self.wallet.write()?.sync_and_save()?;
```

You can remove the scope now